src/features/defi/providers/osmosis/components/OsmosisManager/Lp/Withdraw/components/Withdraw.tsx

```diff
@@ -368,32 +368,30 @@ export const Withdraw: React.FC<WithdrawProps> = ({
           assets,
           marketData,
         })
-        if (underlyingAssetBalances) {
-          trackOpportunityEvent(
-            MixPanelEvents.WithdrawContinue,
-            {
-              opportunity: osmosisOpportunity,
-              fiatAmounts: [
-                underlyingAssetBalances[underlyingAsset0.assetId].fiatAmount,
-                underlyingAssetBalances[underlyingAsset1.assetId].fiatAmount,
-              ],
-              cryptoAmounts: [
-                { assetId: lpAsset.assetId, amountCryptoHuman: formValues.cryptoAmount },
-                {
-                  assetId: underlyingAsset0.assetId,
-                  amountCryptoHuman:
-                    underlyingAssetBalances[underlyingAsset0.assetId].cryptoBalancePrecision,
-                },
-                {
-                  assetId: underlyingAsset1.assetId,
-                  amountCryptoHuman:
-                    underlyingAssetBalances[underlyingAsset1.assetId].cryptoBalancePrecision,
-                },
-              ],
-            },
-            assets,
-          )
-        }
+        trackOpportunityEvent(
```

Comment:
we should be aware that this change could result in mix panel tracking of crypto amount differing from fiat amount or incorrect crypto balance precision. I am not super in tune with tracking and how we are querying data, but I know we have had precision issues, so using default precision of `0` will affect `amountCryptoHuman` tracking

Reply:
Yeah, realistically, this is the same flow as before, though we might be tracking 0s here if the `getUnderlyingAssetIdsBalance` call is wrong for some reason - precision shouldn't change here though? We default to 0 for the `fiatAmount` / `cryptoBalancePrecision` in case of missing assets, regardless of precision

Reply:
Man these names confuse the shit out of me. I read `cryptoBalancePrecision` thinking crypto precision ie. 18.